packages/cli/src/ceramic-daemon.ts

```diff
@@ -76,10 +76,17 @@ export function makeCeramicConfig(opts: DaemonConfig): CeramicConfig {
     return new RotatingFileStream(logPath, true)
   })
 
+  const metricsExporterEnabled = opts.metrics?.metricsExporterEnabled && opts.metrics?.collectorHost;
+  const prometheusExporterEnabled = opts.metrics?.prometheusExporterEnabled && opts.metrics?.prometheusExporterPort;
+
   // If desired, enable metrics
-  if (opts.metrics?.metricsExporterEnabled && opts.metrics?.collectorHost) {
+  if (metricsExporterEnabled && prometheusExporterEnabled) {
+    Metrics.start(opts.metrics.collectorHost, CALLER_NAME, DEFAULT_TRACE_SAMPLE_RATIO, null, true, opts.metrics.prometheusExporterPort);
+  } else if (metricsExporterEnabled) {
     Metrics.start(opts.metrics.collectorHost, CALLER_NAME)
-  }
+  } else if (prometheusExporterEnabled) {
+    Metrics.start('', CALLER_NAME, DEFAULT_TRACE_SAMPLE_RATIO, null, true, opts.metrics.prometheusExporterPort);
```

Comment:
did you already add this param in obervability package

Reply:
yes, it was added in 1.3.0